test/serializing.nim

```diff
@@ -101,6 +101,54 @@ suite "Serialization":
       except: gotException = true
       assert gotException, "Expected exception, got none."
 
+  test "Serialization: Load Hex byte (0xFF)":
```

Comment:
Please remove `Serialization:` from all the names. That is not necessary anymore as 0.14.3 displays the suite name with the test output.